dev/files/policy.yml

```diff
@@ -1 +1,3 @@
 - !user alice
+
+- !variable testvar
```

Comment:
I suspect this dev policy change is leftover from debugging/testing. Is it needed? 

Same comment for the conjur.yml file that was added